gtrending/cli.py

```diff
@@ -0,0 +1,187 @@
+import argparse
+import json
+
+from typing import Dict, Any
+
+from .fetch import (
+    fetch_repos,
+    fetch_developers,
+    languages_list,
+    spoken_languages_list,
+)
+
+
+def main():
+
+    parser = argparse.ArgumentParser('Gtrending')
+    subparser = parser.add_subparsers(dest='command')
+  
+    # Extract common arguments in parents
+    parent_json = argparse.ArgumentParser(add_help=False)
+    parent_json.add_argument(
+        '--json',
+        '-J',
+        action=argparse.BooleanOptionalAction,
```

Comment:
`BooleanOptionalAction` was only added in Python 3.9. Unfortunately, it would be ideal for the library to support up to 3.6, or at least 3.8 and above in my opinion. Could we go with an alternative method for handling the boolean flags?